### ISSUE TITLE
Complete the empty cells in the table

### ISSUE THREAD

**Issue by simonhac** [opennem/opennem-fe#251]
missing data to fill in:

| Note | Comment |
|--------|--------|
| 1 & 2. | these values are currently always showing `$0.00`, but shouldn't be! |
| 3. | for sources: show total GWh, total contribution % (can be over 100%), VWP |
| 4. | for loads: show total contribution % and VWP |
| 5. | for sources less loads: show total contribution (should be 100%?), and VWP |
| 6. | for renewables: show VWP |

<img width="400" alt="image" src="https://github.com/opennem/opennem-fe/assets/20500649/fd13f4e2-b937-4921-872d-298407a73239">

**Comment by aleith** [comment]
Does OpenNEM work out the emission intensity of energy imports? If yes, fantastic! Is this included in the summary data  and graphs for each state's "consumption" as opposed to "generation" which otherwise just removes all exported energy from the emissions I assume?

If no, there's a methodology that they use on CO2maps.de linked to in their FAQ… it leverages some published academic work by some of their contributors.

**Comment by simonhac** [comment]
@chienleng when a fueltech is disabled, its emissions volume (and intensity) shouldn't go to zero — as with the other metrics, the display values shouldn't change.

<img width="350" alt="image" src="https://github.com/user-attachments/assets/566ca6b4-71b0-4296-a867-084ef217508e">

**Comment by simonhac** [comment]
> Does OpenNEM work out the emission intensity of energy imports? If yes, fantastic! Is this included in the summary data and graphs for each state's "consumption" as opposed to "generation" which otherwise just removes all exported energy from the emissions I assume?
> 
> If no, there's a methodology that they use on CO2maps.de linked to in their FAQ… it leverages some published academic work by some of their contributors. Should I make a new issue describing this or I this concern already completely sorted?

@aleith yes, OpenNEM calculates the emissions intensity of energy imports/exports, and yes, this is included in the charts for each state's consumption. eg. you can 'see' the emissions SA is importing from victoria: 

<img width="500" alt="image" src="https://github.com/user-attachments/assets/161728b3-65ae-4da0-9e3e-9021c2be4d6e">

we use the methodology reportedly used by [electricitymaps.com](electricitymaps.com).

**Comment by aleith** [comment]
that's great, @simonhac . I'm familiar with the electricity maps methodology and iterations it went through so, amazing to get this level of accuracy.

**Comment by simonhac** [comment]
@chienleng while we're here, please address this discrepancy:

<img width="600" height="1018" alt="Image" src="https://github.com/user-attachments/assets/846cd398-7ad0-4f89-bccc-df043f73a01e" />
